src/main/kotlin/no/nav/tilleggsstonader/sak/behandling/BehandlingService.kt

```diff
@@ -284,4 +284,9 @@ class BehandlingService(
             ),
         )
     }
+
+    fun utledNesteBehandlingstype(fagsakId: UUID): BehandlingType {
+        val behandlinger = hentBehandlinger(fagsakId)
+        return if (behandlinger.all { it.resultat == BehandlingResultat.HENLAGT }) BehandlingType.FØRSTEGANGSBEHANDLING else BehandlingType.REVURDERING
+    }
```

Comment:
Føles litt unødvendig å hente alle behandlinger for denne type metode, men er kanskje tydligere enn noe à la
```kotlin
val sisteBehandling = repo.finnSiste()
if(sisteBehandling.forrigeBehandlingId != null || resultat != HENLAGT) {
  REVURDERING
} else {
  FØRSTEGANGSBEHANDLING
}
```

Reply:
Kan legge til en test 👍